test/opt/type_manager_test.cpp

```diff
@@ -14,6 +14,7 @@
 
 #include "source/opt/type_manager.h"
 
+#include <iostream>
```

Comment:
Accidental include?

Reply:
Dang. That was probably me.

Reply:
Thank you for clarification. Will remove.